compiler/tests/fixtures/create2_contract.fe

```diff
@@ -0,0 +1,9 @@
+contract Foo:
+    pub def get_my_num() -> u256:
+        return 42
+
+contract FooFactory:
+    pub def create2_foo() -> address:
+        # value and salt
+        foo: Foo = Foo.create2(0, 52)
```

Comment:
I created an issue for the proposed shift to double colon syntax (`Foo::create2(0, 52)`) https://github.com/ethereum/fe/issues/241

Reply:
great! thank you